app/db/migrations/20171016091057-create-product.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
         type: Sequelize.DECIMAL
       },
       category_id: {
-        type: Sequelize.INTEGER
+        type: Sequelize.STRING
```

Comment:
is this going to be a UUID? or what is the reason for changing it?

Reply:
This has nothing to do with UUID, If you check the models you will have a broader sense of what I'm trying to achieve. a product can belong to multiple categories, so basically It require d that the user send an array of category ID. In the model, All I did was to convert that array to a comma-separated string before saving. If there is any performance gain in using UUID then we can switch to that.